doc/nrf/app_dev/device_guides/nrf54l/kmu_provision.rst

```diff
@@ -73,7 +73,22 @@ Once you have an unprovisioned SoC, upload keys to the board by running the foll
 
 * Parameter ``--dev-id`` specifies the interface serial number and should be used if multiple J-link interfaces are connected to the development machine.
 
+* Parameter ``-p" (--policy)`` specifies the policy applied to the given set of keys.
+  You can apply the following options:
+
+      * ``lock-last`` - Uploads the last key as locked,  while the preceding keys are revocable. This option is set by default.
+      * ``revokable`` - Enables revocation for each key.
+      * ``lock`` - Sets all keys to be permanent.
+
 The script generates the public key for each private key and uploads them to your device.
 These public keys generate the verification keys for the application image, which are then used by MCUboot for validation.
 The first key specified in the command is used for signing the application image.
 Currently, the script supports only ED25519 Keys.
+
+For MCUboot notice that it uses one key by default and KMU support in its configuration need to be enabled, otherwise MCUboot fallbacks to compiled in key.
+For provision one key to the board run the following command:
+
+.. parsed-literal::
+   :class: highlight
+
+    west ncs-provision upload -s nrf54l15 -k ed25519.pem -p lock
```

Comment:
They key should be locked even without the `lock` parameter (`lock-last` policy is used by default)

Reply:
I was going to emphasize that provisioning single key should be locked. will remove.